app/uk/gov/hmrc/customs/financials/emailthrottler/services/EmailQueue.scala

```diff
@@ -76,7 +76,7 @@ class EmailQueue @Inject()(mongoComponent: PlayMongoComponent,
     ).toFutureOption().map {
```

Comment:
1. Could you please give a line break before below lines? 

61, 69, 94,102,108

2. Please remove the unused imports
   import org.mongodb.scala.FindObservable
  import com.mongodb.client.model.ReplaceOptions
 import com.mongodb.client.model.Indexes.descending